# Ubuntu: https://bugs.launchpad.net/bugs/364577
# Upstream: http://bugzilla.gnome.org/show_bug.cgi?id=579769
# Patch: http://bugzilla.gnome.org/attachment.cgi?id=133581&action=view
# Description: Glade3 seems not to support marking these strings as
# translatable. This solved the problem.
#
diff --git a/src/nautilus-file-management-properties.ui b/src/nautilus-file-management-properties.ui
--- a/src/nautilus-file-management-properties.ui	
+++ a/src/nautilus-file-management-properties.ui	
@@ -7,13 +7,13 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">Icon View</col>
+        <col id="0" translatable="yes">Icon View</col>
       </row>
       <row>
-        <col id="0">List View</col>
+        <col id="0" translatable="yes">List View</col>
       </row>
       <row>
-        <col id="0">Compact View</col>
+        <col id="0" translatable="yes">Compact View</col>
       </row>
     </data>
   </object>
@@ -23,19 +23,19 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">By Name</col>
+        <col id="0" translatable="yes">By Name</col>
       </row>
       <row>
-        <col id="0">By Size</col>
+        <col id="0" translatable="yes">By Size</col>
       </row>
       <row>
-        <col id="0">By Type</col>
+        <col id="0" translatable="yes">By Type</col>
       </row>
       <row>
-        <col id="0">By Modification Date</col>
+        <col id="0" translatable="yes">By Modification Date</col>
       </row>
       <row>
-        <col id="0">By Emblems</col>
+        <col id="0" translatable="yes">By Emblems</col>
       </row>
     </data>
   </object>
@@ -45,25 +45,25 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">33%</col>
+        <col id="0" translatable="yes">33%</col>
       </row>
       <row>
-        <col id="0">50%</col>
+        <col id="0" translatable="yes">50%</col>
       </row>
       <row>
-        <col id="0">66%</col>
+        <col id="0" translatable="yes">66%</col>
       </row>
       <row>
-        <col id="0">100%</col>
+        <col id="0" translatable="yes">100%</col>
       </row>
       <row>
-        <col id="0">150%</col>
+        <col id="0" translatable="yes">150%</col>
       </row>
       <row>
-        <col id="0">200%</col>
+        <col id="0" translatable="yes">200%</col>
       </row>
       <row>
-        <col id="0">400%</col>
+        <col id="0" translatable="yes">400%</col>
       </row>
     </data>
   </object>
@@ -73,25 +73,25 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">33%</col>
+        <col id="0" translatable="yes">33%</col>
       </row>
       <row>
-        <col id="0">50%</col>
+        <col id="0" translatable="yes">50%</col>
       </row>
       <row>
-        <col id="0">66%</col>
+        <col id="0" translatable="yes">66%</col>
       </row>
       <row>
-        <col id="0">100%</col>
+        <col id="0" translatable="yes">100%</col>
       </row>
       <row>
-        <col id="0">150%</col>
+        <col id="0" translatable="yes">150%</col>
       </row>
       <row>
-        <col id="0">200%</col>
+        <col id="0" translatable="yes">200%</col>
       </row>
       <row>
-        <col id="0">400%</col>
+        <col id="0" translatable="yes">400%</col>
       </row>
     </data>
   </object>
@@ -101,25 +101,25 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">33%</col>
+        <col id="0" translatable="yes">33%</col>
       </row>
       <row>
-        <col id="0">50%</col>
+        <col id="0" translatable="yes">50%</col>
       </row>
       <row>
-        <col id="0">66%</col>
+        <col id="0" translatable="yes">66%</col>
       </row>
       <row>
-        <col id="0">100%</col>
+        <col id="0" translatable="yes">100%</col>
       </row>
       <row>
-        <col id="0">150%</col>
+        <col id="0" translatable="yes">150%</col>
       </row>
       <row>
-        <col id="0">200%</col>
+        <col id="0" translatable="yes">200%</col>
       </row>
       <row>
-        <col id="0">400%</col>
+        <col id="0" translatable="yes">400%</col>
       </row>
     </data>
   </object>
@@ -129,13 +129,13 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">Always</col>
+        <col id="0" translatable="yes">Always</col>
       </row>
       <row>
-        <col id="0">Local Files Only</col>
+        <col id="0" translatable="yes">Local Files Only</col>
       </row>
       <row>
-        <col id="0">Never</col>
+        <col id="0" translatable="yes">Never</col>
       </row>
     </data>
   </object>
@@ -145,13 +145,13 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">Always</col>
+        <col id="0" translatable="yes">Always</col>
       </row>
       <row>
-        <col id="0">Local Files Only</col>
+        <col id="0" translatable="yes">Local Files Only</col>
       </row>
       <row>
-        <col id="0">Never</col>
+        <col id="0" translatable="yes">Never</col>
       </row>
     </data>
   </object>
@@ -161,34 +161,34 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">100 KB</col>
+        <col id="0" translatable="yes">100 KB</col>
       </row>
       <row>
-        <col id="0">500 KB</col>
+        <col id="0" translatable="yes">500 KB</col>
       </row>
       <row>
-        <col id="0">1 MB</col>
+        <col id="0" translatable="yes">1 MB</col>
       </row>
       <row>
-        <col id="0">3 MB</col>
+        <col id="0" translatable="yes">3 MB</col>
       </row>
       <row>
-        <col id="0">5 MB</col>
+        <col id="0" translatable="yes">5 MB</col>
       </row>
       <row>
-        <col id="0">10 MB</col>
+        <col id="0" translatable="yes">10 MB</col>
       </row>
       <row>
-        <col id="0">100 MB</col>
+        <col id="0" translatable="yes">100 MB</col>
       </row>
       <row>
-        <col id="0">1 GB</col>
+        <col id="0" translatable="yes">1 GB</col>
       </row>
       <row>
-        <col id="0">2 GB</col>
+        <col id="0" translatable="yes">2 GB</col>
       </row>
       <row>
-        <col id="0">4 GB</col>
+        <col id="0" translatable="yes">4 GB</col>
       </row>
     </data>
   </object>
@@ -198,13 +198,13 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">Always</col>
+        <col id="0" translatable="yes">Always</col>
       </row>
       <row>
-        <col id="0">Local Files Only</col>
+        <col id="0" translatable="yes">Local Files Only</col>
       </row>
       <row>
-        <col id="0">Never</col>
+        <col id="0" translatable="yes">Never</col>
       </row>
     </data>
   </object>
@@ -214,13 +214,13 @@ 
     </columns>
     <data>
       <row>
-        <col id="0">Always</col>
+        <col id="0" translatable="yes">Always</col>
       </row>
       <row>
-        <col id="0">Local Files Only</col>
+        <col id="0" translatable="yes">Local Files Only</col>
       </row>
       <row>
-        <col id="0">Never</col>
+        <col id="0" translatable="yes">Never</col>
       </row>
     </data>
   </object>
